x/committee/keeper/proposal.go

```diff
@@ -251,13 +251,32 @@ func (k Keeper) enactProposal(ctx sdk.Context, proposal types.Proposal) error {
 // CloseProposal deletes proposals and their votes, emitting an event denoting the final status of the proposal
 func (k Keeper) CloseProposal(ctx sdk.Context, proposal types.Proposal, outcome types.ProposalOutcome) {
 
+	proposalTally := types.ProposalPollingStatus{}
+	committee, found := k.GetCommittee(ctx, proposal.CommitteeID)
+	if found {
+		switch com := committee.(type) {
+		case types.MemberCommittee:
+			currVotes := k.TallyMemberCommitteeVotes(ctx, proposal.ID)
+			possibleVotes := sdk.NewDec(int64(len(com.Members)))
+			memberPollingStatus := types.NewProposalPollingStatus(proposal.ID, currVotes,
+				currVotes, possibleVotes, com.VoteThreshold, sdk.Dec{Int: nil})
+			proposalTally = memberPollingStatus
+		case types.TokenCommittee:
+			yesVotes, _, currVotes, possibleVotes := k.TallyTokenCommitteeVotes(ctx, proposal.ID, com.TallyDenom)
+			tokenPollingStatus := types.NewProposalPollingStatus(proposal.ID, yesVotes,
+				currVotes, possibleVotes, com.VoteThreshold, com.Quorum)
+			proposalTally = tokenPollingStatus
+		}
+	}
+
 	k.DeleteProposalAndVotes(ctx, proposal.ID)
 
 	ctx.EventManager().EmitEvent(
 		sdk.NewEvent(
 			types.EventTypeProposalClose,
 			sdk.NewAttribute(types.AttributeKeyCommitteeID, fmt.Sprintf("%d", proposal.CommitteeID)),
 			sdk.NewAttribute(types.AttributeKeyProposalID, fmt.Sprintf("%d", proposal.ID)),
+			sdk.NewAttribute(types.AttributeKeyProposalTally, proposalTally.String()),
```

Comment:
Parsing out the results of a proposal from this attribute will be difficult for clients since we use a custom `String` function.
I think it's reasonably likely that someone will want to parse the results and display them somewhere, so we should probably change this.
However storing the results sounds like a lot of work. So I propose just marshalling `proposalTally` to a json string in the attribute. Then it should be easy enough the fetch the results from the block the proposal closed in. @denalimarsh @karzak thoughts?

I'd use `k.cdc.MarshalJSON`, and if it errors probably just log the error. `CloseProposal` is called in the begin blocker so we don't really want to panic and halt the chain over some string formatting.

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/919/commits/afa5818c1a7ef55529d892006f8d2e7bf1e1c648